src/benchmarks/micro/corefx/System.Memory/ReadOnlySequence.cs

```diff
@@ -0,0 +1,182 @@
+        }
+
+        [Benchmark]
+        public void Start_And_EndPosition()
+        {
+            _sequence.Slice(0, _end);
+        }
+
+        [Benchmark]
+        public void StartPosition_And_Length()
+        {
+            _sequence.Slice(_start, 3);
+        }
+
+        [Benchmark]
+        public void StartPosition_And_EndPosition()
+        {
+            _sequence.Slice(_start, _end);
+        }
+
+        [Benchmark]
+        public ReadOnlySequence<byte> RepeatSlice()
+        {
+            var localSequence = _sequence.Slice(0, 10000);
+            localSequence = localSequence.Slice(0, 5000);
+            localSequence = localSequence.Slice(0, 2500);
+            localSequence = localSequence.Slice(0, 1250);
+            localSequence = localSequence.Slice(0, 625);
+            return localSequence;
+        }
+
+        // Add multiple calls for Slice(startPosition, endPosition)
+        [Benchmark]
+        public ReadOnlySequence<byte> RepeatSlice_StartPosition_And_EndPosition()
+        {
+            var localSequence = _sequence.Slice(_start, _end);
+            localSequence = localSequence.Slice(_start, localSequence.End);
+            localSequence = localSequence.Slice(_start, localSequence.End);
+            localSequence = localSequence.Slice(_start, localSequence.End);
+            localSequence = localSequence.Slice(_start, localSequence.End);
+            return localSequence;
+        }
+
+        // MultiSegment Benchmarks
+        [Benchmark]
+        public void MS_StartPosition()
+        {
+            _multiSegmentSequence.Slice(_ms_start);
+        }
+
+        [Benchmark]
+        public void MS_Start()
+        {
+            _multiSegmentSequence.Slice(0);
+        }
+
+        [Benchmark]
+        public void MS_Start_And_Length()
+        {
+            _multiSegmentSequence.Slice(0, 10000);
+        }
+
+        [Benchmark]
+        public void MS_Start_And_EndPosition()
+        {
+            _multiSegmentSequence.Slice(0, _ms_end);
+        }
+
+        [Benchmark]
+        public void MS_StartPosition_And_Length()
+        {
+            _multiSegmentSequence.Slice(_ms_start, 3);
+        }
+
+        [Benchmark]
+        public void MS_StartPosition_And_EndPosition()
+        {
+            _multiSegmentSequence.Slice(_ms_start, _ms_end);
+        }
+
+        [Benchmark]
+        public ReadOnlySequence<byte> MS_RepeatSlice()
+        {
+            var localSequence = _multiSegmentSequence.Slice(0, 10000);
+            localSequence = localSequence.Slice(0, 5000);
+            localSequence = localSequence.Slice(0, 2500);
+            localSequence = localSequence.Slice(0, 1250);
+            localSequence = localSequence.Slice(0, 625);
+            return localSequence;
+        }
+
+        // Add multiple calls for Slice(startPosition, endPosition)
+        [Benchmark]
+        public ReadOnlySequence<byte> MS_RepeatSlice_StartPosition_And_EndPosition()
```

Comment:
Can you share the result of this benchmark (along with some of the other missing ones)? I only saw a few mentioned in https://github.com/dotnet/corefx/pull/38431#issuecomment-505678081.